app/helpers/object_link_helper.rb

```diff
@@ -160,7 +162,7 @@ def location_search_links(name)
   def search_link_to(site_symbol, search_string)
     return unless (url = LOCATION_SEARCH_URLS[site_symbol])
 
-    link_to(site_symbol.to_s.titlecase, url << search_string)
+    link_to(site_symbol.to_s.titlecase, "#{url}#{search_string}")
```

Comment:
Uses a frozen string. An alternative, with a mutable string:
```ruby
link_to(site_symbol.to_s.titlecase, url + search_string)
```

Reply:
Conceptually I prefer this to be frozen if it can.  I would be surprised if we actually later modify this string.